matlab/FinAngles.m

```diff
@@ -0,0 +1,93 @@
+
+function [f1,f2,f3,f4]=FinAngles(roll_alpha,pitch_alpha,yaw_alpha)
+    %Sets maximum accelerations used for normalization
+    roll_max=200;
+    pitch_max=200;
+    yaw_max=100;
+    
+    %Fin/Canard angles for neutral stability (fin angle needed for CP=CG)
+    fin_eq=0;
+    canard_eq=41.418;
```

Comment:
Canards need to be deployed at 41.4 degrees to be neutrally stable? This doesn't sound right, the rear fins are much larger

Reply:
Nah, its right. The geometry I chose was such that the CG was closer to tail than CP. Thus it would point nose up if dropped

Reply:
Then i think we should choose a CG that has both of these `_eq` params be < 10
otherwise, we're losing out on potential drag. Remember, we want this phase to be like an airbrake mode.